.npmignore

```diff
@@ -1 +1,2 @@
 yarn.lock
+*.tgz
```

Comment:
Do we use `.tgz` extension? 

Also, let's ignore sources and Typescript configs, as we don't need them in npm registry

```suggestion
/src
tsconfig.json
```